unit-tests/dds/d405.py

```diff
@@ -224,41 +225,42 @@ def depth_stream():
 
 def ir_stream_profiles():
     return [
-        dds.video_stream_profile( 25, dds.stream_format("Y16"),  1288, 808 ),
-        dds.video_stream_profile( 15, dds.stream_format("Y16"),  1288, 808 ),
-        dds.video_stream_profile( 30, dds.stream_format("mono8"), 1280, 720 ),
-        dds.video_stream_profile( 15, dds.stream_format("mono8"), 1280, 720 ),
-        dds.video_stream_profile( 5,  dds.stream_format("mono8"), 1280, 720 ),
-        dds.video_stream_profile( 90, dds.stream_format("mono8"), 848, 480 ),
-        dds.video_stream_profile( 60, dds.stream_format("mono8"), 848, 480 ),
-        dds.video_stream_profile( 30, dds.stream_format("mono8"), 848, 480 ),
-        dds.video_stream_profile( 15, dds.stream_format("mono8"), 848, 480 ),
-        dds.video_stream_profile( 5,  dds.stream_format("mono8"), 848, 480 ),
-        dds.video_stream_profile( 90, dds.stream_format("mono8"), 640, 480 ),
-        dds.video_stream_profile( 60, dds.stream_format("mono8"), 640, 480 ),
-        dds.video_stream_profile( 30, dds.stream_format("mono8"), 640, 480 ),
-        dds.video_stream_profile( 15, dds.stream_format("mono8"), 640, 480 ),
-        dds.video_stream_profile( 5,  dds.stream_format("mono8"), 640, 480 ),
-        dds.video_stream_profile( 90, dds.stream_format("mono8"), 640, 360 ),
-        dds.video_stream_profile( 60, dds.stream_format("mono8"), 640, 360 ),
-        dds.video_stream_profile( 30, dds.stream_format("mono8"), 640, 360 ),
-        dds.video_stream_profile( 15, dds.stream_format("mono8"), 640, 360 ),
-        dds.video_stream_profile( 5,  dds.stream_format("mono8"), 640, 360 ),
-        dds.video_stream_profile( 90, dds.stream_format("mono8"), 480, 270 ),
-        dds.video_stream_profile( 60, dds.stream_format("mono8"), 480, 270 ),
-        dds.video_stream_profile( 30, dds.stream_format("mono8"), 480, 270 ),
-        dds.video_stream_profile( 15, dds.stream_format("mono8"), 480, 270 ),
-        dds.video_stream_profile( 5,  dds.stream_format("mono8"), 480, 270 ),
-        dds.video_stream_profile( 90, dds.stream_format("mono8"), 424, 240 ),
-        dds.video_stream_profile( 60, dds.stream_format("mono8"), 424, 240 ),
-        dds.video_stream_profile( 30, dds.stream_format("mono8"), 424, 240 ),
-        dds.video_stream_profile( 15, dds.stream_format("mono8"), 424, 240 ),
-        dds.video_stream_profile( 5,  dds.stream_format("mono8"), 424, 240 )
+        dds.video_stream_profile( 25, dds.stream_format("Y12I"),  1288, 808 ),
```

Comment:
These are per IR stream (left/right), so should technically be already de-interlaced?